packages/dds/matrix/src/matrix.ts

```diff
@@ -529,6 +532,36 @@ export class SharedMatrix<T = any>
 
 		// Only need to store it in the snapshot if we have switched the policy already.
 		if (this.setCellLwwToFwwPolicySwitchOpSeqNumber > -1) {
+			const cellLastWriteTrackerToSummarize = new SparseArray2D<CellLastWriteTrackerItem>();
+			let count = 100000;
```

Comment:
Not sure what it is for/ I see it only decremented, but never read from

Reply:
I think you likely wanted to say - we will scan that many rows?
Might be better to be count of cells, not rows, as table theoretically could have a lot of columns

Reply:
That said, given that you do not change it in place, but rather create a copy (cellLastWriteTrackerToSummarize), you can't bail out sooner, right?

Reply:
yeah, there is not in place freeing up the cells. I don't think we can do it partially and also cannot do it completely for bigger matrices. Closing this PR for now.